### PR TITLE
Clarify first task for lesson 04

### PR DESCRIPTION
**Pull request by kuszcjan**
Some folks have problem with mocking input()

### CODE REVIEW

**Comment by jedzej** [comment]
To do what is requested they'd need to construct a list, add elements to it, iterate over it to print output in main and use list related assertions in test case. They don't know how to do it so far, since lists will be introduced in the next lesson. My proposal is to keep this this lesson out of it, but **move this exercise to next lesson** as very good example of tests improvements by involving richer python features.

In this lesson I'd focus on testing collatz function mainly. Let them pick representative test data covering typical cases, corner cases, invalid data etc. What do you think?

**Comment by jedzej** [comment]
@kuszcjan I've forked and extended your snippet to additionally catch print: https://gist.github.com/jedzej/c9129558e87ec5d0b37405afd0ce6f46
With this it should be possible for them to test collatz without understanding lists:)

File: course/lesson_04_unit_testing/README.md [hunk 1]
```diff
@@ -5,10 +5,11 @@
 
 ### practice projects
 
-1. Write unittest.TestCase tests (2-3) for (The Collatz Sequence) function. 
-Define test data for tests in setUp method. 
+1. Modify (The Collatz Sequence) so that its main function takes an argument and return list with all numbers.
```
Comment (adziu) [inline course/lesson_04_unit_testing/README.md]: add "from lesson 03"

File: course/lesson_04_unit_testing/README.md [hunk 2]
```diff
@@ -5,10 +5,11 @@
...
+Write `unittest.TestCase` tests (2-3) for it.
```
Comment (adziu) [inline course/lesson_04_unit_testing/README.md]: This is supposed to mean 2 or 3? Give a specific test to implement, so that it's clear how it should be done. For example:
* test that it crashes on input 'not a number'
* test that for input 4 it retuns 2,

**Comment by kuszcjan** [comment]
Removed monkey patching part. Need to merge this one or #225
